vue/src/views/SmartFlow.vue

```diff
@@ -0,0 +1,183 @@
+            />
+          </template>
+          <!-- <template #item.actions="{ item }">
+            <v-icon color="primary" @click="startRun(item)">
+              mdi-play
+            </v-icon>
+            <v-icon color="error" @click="stopRun(item)">
+              mdi-stop
+            </v-icon>
+          </template> -->
+          <template #[`item.state`]="{ item }">
+            <v-chip
+              :color="stateColor(item.state)"
+              dark
+            >
+              {{ item.state }}
+              <v-progress-circular
+                v-if="item.state === 'running'"
+                indeterminate
+                :size="20"
+                class="ml-2"
+              />
+            </v-chip>
+          </template>
+        </v-data-table>
+        <v-btn
+          color="primary"
+          @click="showNewDagRunDialog"
+        >
+          Start New DAG Run
+        </v-btn>
+        <v-dialog
+          v-model="newDagRunDialog"
+          max-width="500"
+        >
+          <v-card>
+            <v-card-title>Enter Region</v-card-title>
+            <v-card-text>
+              <v-text-field
+                v-model="newDagRunTitle"
+                label="DAG Run Title"
+              />
+            </v-card-text>
+            <v-card-text>
+              <v-textarea
+                v-model="newDagRunConf"
+                label="conf"
+              />
+            </v-card-text>
+            <v-card-actions>
+              <v-btn
+                color="primary"
+                @click="startNewRun"
+              >
+                Start
+              </v-btn>
+              <v-btn
+                color="error"
+                @click="newDagRunDialog = false"
+              >
+                Cancel
+              </v-btn>
+            </v-card-actions>
+          </v-card>
+        </v-dialog>
+      </v-card-text>
+    </v-card>
+  </v-container>
+</template>
+
+<script setup lang="ts">
+import { onMounted, ref, watch } from 'vue';
+import { debounce } from 'lodash';
+import { SmartflowService } from '../client';
+
+interface DagRun {
+  dag_id: string;
+  dag_run_id: string;
+  data_interval_end: string;
+  data_interval_start: string;
+  end_date: string;
+  execution_date: string;
+  external_trigger: boolean;
+  last_scheduling_decision: string;
+  logical_date: string;
+  note: string | null;
+  run_type: string;
+  start_date: string;
+  state: string;
+}
+
+const headers = [
+  { title: 'DAG ID', key: 'dag_id' },
+  { title: 'DAG Run ID', key: 'dag_run_id' },
+  { title: 'Status', key: 'state' },
+  { title: 'Actions', key: 'actions' },
+];
+
+const dag = ref('');
+const dagRuns = ref<DagRun[]>([]);
+const itemsPerPage = ref(10);
+const totalItems = ref(0);
+const loading = ref(false);
+
+const dagSearch = ref('');
+const dagResults = ref<string[]>([]);
+
+const newDagRunDialog = ref<boolean>(false);
+const newDagRunTitle = ref<string>('');
+const newDagRunConf = ref<string>('');
```

Comment:
could this be defaulted to a sensable value at least for the demo?  Something like the { "regionId": "KR_R001"} so we have some idea of what it expects.